mod-orders/mod-orders.postman_collection.json

```diff
@@ -0,0 +1,967 @@
+						"header": [
+							{
+								"key": "X-Okapi-Tenant",
+								"value": "{{xokapitenant}}"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "x-okapi-token",
+								"value": "{{xokapitoken}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\n}"
+						},
+						"url": {
+							"raw": "{{protocol}}://{{url}}:{{okapiport}}/orders",
+							"protocol": "{{protocol}}",
+							"host": [
+								"{{url}}"
+							],
+							"port": "{{okapiport}}",
+							"path": [
+								"orders"
+							]
+						},
+						"description": "GET /rtac/ requests that return 200"
+					},
+					"response": []
+				},
+				{
+					"name": "POST /orders - create complete order",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "eb2052a4-e388-4860-80be-ea96976bcf21",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "3bf00f6d-8798-4764-a256-2d360ea02876",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									" ",
+									"pm.environment.set(\"order_id\", jsonData.id);",
+									"",
+									"pm.test(\"Status code is 201\", function () {",
+									"    pm.response.to.have.status(201);",
+									"});",
+									"",
+									"pm.test(\"po_lines exist\", function () {",
+									"    pm.expect(jsonData.po_lines).to.have.lengthOf(1);",
+									"});",
+									"",
+									"pm.test(\"Each order has these optional fields\", function(){",
+									"    pm.expect(jsonData.id).to.exist;",
+									"    pm.expect(jsonData.approved).to.exist;",
+									"    pm.expect(jsonData.po_number).to.exist;",
+									"    pm.expect(jsonData.notes).to.exist;",
+									"    pm.expect(jsonData.total_items).to.exist;",
+									"    pm.expect(jsonData.vendor).to.exist;",
+									"});",
+									"",
+									"// pm.test(\"Validate schema\", function () {",
+									"//      tv4.addSchema(\"holding.json\", JSON.parse(pm.variables.get(\"schema_holding_content\")));",
+									"//     pm.expect(tv4.validate(jsonData, JSON.parse(pm.environment.get(\"schema_holdings_content\")))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
+									"//     pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
+									"// });",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "X-Okapi-Tenant",
+								"value": "{{xokapitenant}}"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "x-okapi-token",
+								"value": "{{xokapitoken}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
```

Comment:
I believe, this request needs some modifications, po_number, and all the UUID fields are supposed to be populated after we post an order, we shouldn't be sending them in the POST request

Reply:
@craigmcnally https://s3.amazonaws.com/foliodocs/api/mod-orders/order.html#orders_post does this example need correction?